Inseok_45/controllers/userController.js

```diff
@@ -1,17 +1,22 @@
+const jwt = require('jsonwebtoken');
 const userService = require('../services/userService');
+const { emailValidationCheck, passwordValidationCheck } = require('../utils/validation-check');
+
 
 const signUp = async ( req, res ) => {
     try {
         const { firstName, lastName, email, phoneNumber, age, userName, password } = req.body;
-
+        
         if( !firstName || !lastName || !email || !phoneNumber || !age || !userName || !password)
             return res.status(400).json({ 
                 message: "Cannot Sign Up" 
```

Comment:
```suggestion
                message: "KEY_ERROR" 
```
예외 상황에 맞는 에러 메세지를 명확히 작성해주세요!